client/spa/js/instructor/spec/instructor.model.spec.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+/*
+global jasmine, describe, it, expect, beforeEach, afterEach, xdescribe, xit,
+spyOn
+*/
+// Get the code you want to test
+var Model = require('../instructor.model');
+
+// Test suite
+console.log('test instructor.model');
+describe('Instructor model ', function(){
+  var model;
+
+  describe('when creating a new model ', function(){
+    beforeEach(function(){
+      model = new Model();
+    });
+
+    xit('initializes with custom logic', function(){
+    });
```

Comment:
Looks like in the model's initialize you are listening to a 'change' event. You may want to put that in the test or eliminate that listener.